src/lib/api/postsFetcher.ts

```diff
@@ -61,7 +67,7 @@ query GetPosts($category: String, $tag: String) {
       },
       body: JSON.stringify({
         query: postsQuery,
-        variables: { category, tag },
+        variables: { category, tag, author },
       }),
       cache: "force-cache",
```

Comment:
What are the implications of using force-cache here?

Reply:
from the official website :  when using force-cache, Next.js looks for a matching request in its Data Cache. So if it finds one it will not resend the fetch request. Thus reducing unnecessary fetch requests.